packages/webapp/src/utils/ocrDemo.ts

```diff
@@ -0,0 +1,40 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+/**
+ * Demo Code
+ * Needs to be clean up in next sprint
+ */
+import { imageOcrData } from './demoData'
+
+const callOCR = async (file) => {
+	const formdata = new FormData()
+	let ocrResult = null
+	formdata.append('file', file)
+
+	const requestOptions = {
+		method: 'POST',
+		body: formdata
+	}
+	ocrResult = await fetch('https://cbo-ops-suite.azurewebsites.net/api/ocr', requestOptions)
+	return ocrResult
+}
+
+const localFile = async (file) => {
+	const request = await callOCR(file)
+	const result = []
+	for (const { key, value, confidence } of request) {
```

Comment:
I'm already doing this in the Azure Function, you can consume the result directly!

Reply:
removed